### PR TITLE
fix: profile design

### PR DESCRIPTION
**Pull request by shinevue**

### CODE REVIEW

File: client/src/features/settings/profilesettings/index.js [hunk 1]
```diff
@@ -80,55 +80,59 @@ function ProfileSettings() {
...
-            options={["", ...cates]}
-            onChange={(e) => setHandleTag(e)}
-          />
-        </div>
-        <div className="flex items-center">
-          <div
-            className={`form-control w-11/12 m-2 flex sm:flex-row flex-col input border-2`}
-          >
-            {tag.map((value, index) => (
+          <div>
```
Comment (oddcommitking) [inline client/src/features/settings/profilesettings/index.js:83]: Good work.

**Review by oddcommitking — APPROVED** [review]

File: client/src/features/settings/profilesettings/index.js (hunk 1 above)
Comment (shinevue) [reply client/src/features/settings/profilesettings/index.js:83]: Thank you

**Review by shinevue — COMMENTED** [review]
Thank you

File: client/src/features/settings/profilesettings/index.js [hunk 2]
```diff
@@ -80,55 +80,59 @@ function ProfileSettings() {
...
+              label={"Category"}
+              class="w-full "
+              options={["", ...cates]}
+              onChange={(e) => setHandleTag(e)}
+            />
+            <div className="flex">
               <div
-                key={index}
-                className="flex sm:flex-row flex-col items-center m-1 w-max border-2 mr-2"
+                className={`form-control w-11/12 m-2 flex sm:flex-row flex-col input border-2`}
```
Comment (oddcommitking) [inline client/src/features/settings/profilesettings/index.js:108]: Match responsive

**Review by oddcommitking — APPROVED** [review]

File: client/src/features/settings/profilesettings/index.js (hunk 2 above)
Comment (shinevue) [reply client/src/features/settings/profilesettings/index.js:108]: I got it

**Review by shinevue — COMMENTED** [review]
Hi, reviewed solved?

**Review by oddcommitking — APPROVED** [review]
Reviewed

**Pull request closed without merging**